src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -5,5 +5,5 @@
 import java.util.List;
 
 public class Storage {
-    public static final List<User> people = new ArrayList<>();
+    public static final List<User> PEOPLE = new ArrayList<>();
```

Comment:
you should not modify existing classes - there is no need for that

Reply:
After "mvn clean package" I received:
Name 'people' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]
So I changed it. How can I solve this problem in another way?